examples/ampi/privatisation_tutorial/nm_globals.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
Is this copy of nm_globals.sh required?

Reply:
This copy is placed there for convenience. I could get rid of it if needed.

Reply:
Would it make sense to replace the copy with a symlink or so?

Reply:
That's a good point. I will make that change. So will this pull request need to closed?

Reply:
No need to close this pull request, you can just push another commit to the branch to update it.

Reply:
Made the changes to nm_globals.sh. Please verify.

Reply:
looks good to me